tower/src/balance/p2c/service.rs

```diff
@@ -107,6 +107,11 @@ where
     pub fn len(&self) -> usize {
         self.services.len()
     }
+
+    /// Returns whether or not the balancer is empty.
+    pub fn is_empty(&self) -> bool {
+        self.len() == 0
```

Comment:
I would expect this to trigger clippy. Should this be:
```suggestion
        self.services.is_empty()
```

Reply:
I'm not sure if ReadyCache actually has an `is_empty`, but it probably should if it does not...

Reply:
Uh good catch. Fixed in 5dc604458ccf7